File: C_Programming/Day_3/program11.c
// Description : Write a program to check whether the number is even or odd.

# include<stdio.h>
#include<stdbool.h>

bool CheckEven(int iNo)
{
    bool bFlag = false;

    if(iNo % 2 == 0)
    {
        bFlag = true;
    }

    return bFlag;

}


int main()
{
    int iValue = 0;
    bool bRet = false;
    printf("Enter the number : ");
    scanf("%d",&iValue);

    bRet = CheckEven(iValue);

    if(bRet == true)
    {
        printf("It is an even number...\n");
    }
    else
    {
        printf("It is an odd number...\n");
    }


    return 0;
}

File: Python_Programming/Day_3/program11.py
# Description : Write a program to check whether the number is even or odd.

def CheckEven(No):
    Flag = False

    if(No % 2 == 0):
        Flag = True

    return Flag

def main():

    Value = int(input("Enter the number : "))

    Ret = CheckEven(Value)

    if(Ret == True):
        print("It's an even number...")
    else:
        print("It's an odd number...")
    

if __name__ == "__main__":
    main()